src/components/Icon/Icons.ts

```diff
@@ -0,0 +1,722 @@
+import AccessibilityNewSvg from 'uswds/dist/img/usa-icons/accessibility_new.svg?react'
```

Comment:
Note the query param `?react` at the end of each import. That is what's telling webpack to load these files using `svgr`, and all other SVGs are still loaded using the asset loader (so SVGs imported as images or in CSS should not break)

Reply:
That's neat! Total bikeshed, but would using `?svgr` maybe make it more obvious about what is going on? Feel free to ignore this comment :)

Reply:
OH I like that idea! tbh I got this from following an example elsewhere and at first didn't understand the use of `?react`. I will change it to `?svgr` as I think that is much clearer.